INSTALL.rst

```diff
@@ -11,24 +11,39 @@ Prerequisite Software
 ---------------------
 
 HaaS requires a number of packages available through the CentOS RPM
-repositories, as well as the EPEL repository. EPEL can be enabled via:
-
-::
+repositories, as well as the EPEL repository. EPEL can be enabled via::
 
     yum install epel-release
 
-Then, the rest of the packages can be installed via:
-
-::
+Then, the rest of the packages can be installed via::
 
-    yum install libvirt bridge-utils ipmitool telnet sqlite httpd mod_wsgi python-pip qemu-kvm python-virtinst
+    yum install libvirt bridge-utils ipmitool telnet httpd mod_wsgi python-pip qemu-kvm python-virtinst virt-install python-psycopg2
 
 In addition, HaaS depends on a number of python libraries. Many of these are
 available as RPMs as well, but we recommend installing them with pip, since
 this will install the versions that HaaS has been tested with.  This is done
 automatically by the instructions below.
 
-The HaaS software can then be installed by running:
+Setting Up HaaS Database
+------------------------
+
+The only DBMS currently supported for production use is PostgreSQL. (SQLite is
+supported for development purposes *only*). There are many guides on the web
+which describe setting up PostgreSQL; in these instructions we assume that
+you've done so successfully.
+
+You need to create a postgresql user and database for HaaS. once you've done so,
+the ``uri`` option in ``haas.cfg``'s ``database`` must be set to::
+
+        postgresql://<user>:<password>@<address>/<dbname>
+
+Where ``<user>`` is the name of the postgres user you created, ``<password>`` is
+its password, ``<dbname>`` is the name of the database you created, and
+``<address>`` is the address which haas should use to connect to postgres (In a
+typical default postgres setup, the right value is ``localhost``.
+
+
+The HaaS software itself can then be installed by running:
 
 ::
```

Comment:
Probably standardize on lines ending with `::`, rather than `:` and then a standalone `::`---we're already doing this in several other places in this patch set.